urbandict-search/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM pytorch/pytorch:latest
+
+WORKDIR /
+
+ENV DATA_DIR /
+
+COPY . /
+
```

The data file should be copied into the docker so that `prepare_data.py` can be executed properly.
```suggestion
COPY urban-dictionary-words-dataset.zip ${DATA_PATH}
```